hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacityScheduler.java

```diff
@@ -1737,6 +1737,10 @@ private CSAssignment allocateContainerOnSingleNode(
 
   private void allocateFromReservedContainer(FiCaSchedulerNode node,
       boolean withNodeHeartbeat, RMContainer reservedContainer) {
+    if(reservedContainer == null){
+      LOG.info("reservedContainer is null,that may be unreserved by the proposal judgment thread");
```

Comment:
Maybe it is better for waring level log.

Reply:
> Maybe it is better for waring level log.

Thanks for your review,I have updated the patch.